FluidSynthConfig.cmake.in

```diff
@@ -0,0 +1,11 @@
+# for the find_dependency() macro:
+# include(CMakeFindDependencyMacro)
+# it has the same syntax as find_package:
+# find_dependency(MYDEP REQUIRED)
+
+# define variables for configuration options:
+# set(network-enabled @enable-network@)
+
```

Comment:
How about
```suggestion

@PACKAGE_INIT@

```

Reply:
That requires the template to be processed by `configure_package_config_file()`